packages/getting-started/src/browser/getting-started-preferences.ts

```diff
@@ -0,0 +1,39 @@
+// *****************************************************************************
+// Copyright (C) 2023 Ericsson and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { interfaces } from '@theia/core/shared/inversify';
+import { nls } from '@theia/core/lib/common/nls';
+import { PreferenceSchema } from '@theia/core/lib/common/preferences/preference-schema';
+import { PreferenceContribution } from '@theia/core/lib/browser';
+
+export namespace GettingStartedPreferences {
+    export const alwaysShowWelcomePage = 'welcome.alwaysShowWelcomePage';
+}
+
+export const welcomePreferenceSchema: PreferenceSchema = {
+    'type': 'object',
+    'properties': {
+        'welcome.alwaysShowWelcomePage': {
+            type: 'boolean',
+            description: nls.localizeByDefault('Show welcome page on startup'),
+            default: true
+        }
+    }
+};
+
+export function bindWelcomePreference(bind: interfaces.Bind): void {
+    bind(PreferenceContribution).toConstantValue({ schema: welcomePreferenceSchema });
```

Comment:
In order to make schemas extensible for downstream adopters, we should follow the design from https://github.com/eclipse-theia/theia/pull/9883.
You can use existing `*-preferences.ts` files to be inspired:

https://github.com/eclipse-theia/theia/blob/6607185e37361b7251a5ef3df3016348bfa75fc9/packages/workspace/src/browser/workspace-preferences.ts#L50-L58